docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -504,7 +504,7 @@ The keys may be in a different format (PKCS#1 or PKCS#12); in that case, it may
 The following command creates a pkcs8 key file.
 
 ```bash
-$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes​
+$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
```

Comment:
Making a quick suggestion so that this is more in keeping with our style:

```suggestion
openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes
```